portfolio/src/main/webapp/gallery.html

```diff
@@ -19,11 +19,115 @@
         </nav>
       </div>
 
-      <h1 id="pagename">Himani's Image Gallery</h1>
-
-      <div class="images">
-          <!-- TODO: Add images here -->
+      <div class="gallery-area">
+        <h1 id="pagename">Himani's Image Gallery</h1>
+        <p>I love to travel! Click on an image below to see a description.</p>
+        <div class="image-container">
+          <div class="image" onclick="showCaption(1)">
```

Comment:
I like this trick you've used with passing an index, but that may get hard to keep track of if you're reordering things later. I'm going to recommend a fix in the JS that will allow you to just call `showCaption()`